docs/rest/Data/CustomSearchPatientsAdditional.json

```diff
@@ -0,0 +1,81 @@
+{
+    "resourceType": "Bundle",
+    "type": "batch",
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "Patient",
+                "extension": [
+                    {
+                        "url": "http://mycompany.org/fhir/StructureDefinition/test-bool",
```

Comment:
What do these extra examples show? I don't see a search parameter for them or examples using them.

Reply:
Good catch - updated to be more obvious that the "extra resource" is to show null behavior without `:missing`.